src/components/sendV2/summary/summary.js

```diff
@@ -71,46 +127,51 @@ class Summary extends React.Component {
     }
   }
 
-  amountAndFee() {
-    return fromRawLsk(toRawLsk(this.props.fields.amount.value) + fees.send);
-  }
-
   prevStep() {
     Piwik.trackingEvent('Send_Summary', 'button', 'Previous step');
-    this.props.prevStep({ fields: { ...this.props.fields } });
+    this.props.prevStep({ ...this.props.fields });
   }
 
   nextStep() {
     Piwik.trackingEvent('Send_Summary', 'button', 'Next step');
-    this.props.sent({
-      account: this.props.account,
-      recipientId: this.props.fields.recipient.address,
-      amount: this.amountAndFee(),
-      data: this.props.fields.reference.value,
-      passphrase: this.props.account.passphrase,
-      secondPassphrase: this.state.secondPassphrase.value,
+    this.submitTransaction();
+    this.props.nextStep({
+      fields: {
+        ...this.props.fields,
+        hwTransactionStatus: false,
+        isHardwareWalletConnected: false,
+      },
     });
-    this.props.nextStep({ fields: { ...this.props.fields } });
   }
 
   render() {
-    const { secondPassphrase } = this.state;
-    const isBtnDisabled = secondPassphrase.hasSecondPassphrase
+    const { secondPassphrase, isHardwareWalletConnected } = this.state;
+    let isBtnDisabled = secondPassphrase.hasSecondPassphrase
```

Comment:
Can't you just do:
`secondPassphrase.hasSecondPassphrase && secondPassphrase.isValid !== '' && !secondPassphrase.isValid`
Looks better than a ternary that just puts `false`, IMO

Reply:
good, fixed